recipe/meta.yaml

```diff
@@ -14,10 +14,11 @@ source:
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed meson
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  noarch: python
 
 requirements:
-  build:
+  host:
     - python >=3.5
```

Comment:
I thought the `# skip [py2k]` should be preferred?

Reply:
For `noarch` packages, this is preferred (and the `skip` doesn't work).

Reply:
`noarch: python` does the magic to move things into `lib/pythonX.Y/site-packages/` and compile `.pyc`s itself. If you're not / can't use `noarch: python`, then you need to build for each python version separately and do `skip`s for ones you don't want.